gamemode/modules/base/sh_util.lua

```diff
@@ -105,24 +105,27 @@ function DarkRP.findPlayers(info)
         -- Playerinfo is always to be treated as UserID when it's a number
         -- otherwise people with numbers in their names could get confused with UserID's of other players
         if tonumber(PlayerInfo) then
-            if IsValid(Player(PlayerInfo)) and not found[Player(PlayerInfo)] then
-                found[Player(PlayerInfo)] = true
+            local foundPlayer = Player(PlayerInfo)
+            if IsValid(foundPlayer) and not found[foundPlayer] then
+                found[foundPlayer] = true
                 players = players or {}
-                table.insert(players, Player(PlayerInfo))
+                table.insert(players, foundPlayer)
             end
             continue
         end
 
+        local stringPlayerInfo = string.lower(tostring(PlayerInfo))
         for _, v in ipairs(pls) do
-            -- Prevend duplicates
+            -- Prevent duplicates
             if found[v] then continue end
+            local steamId = v:SteamID()
```

Comment:
`SteamID` was called twice per iteration, changing it to once and storing in a variable ain't much, but it also looks a bit nicer in the already long if-statement below.